### DISCUSSION TITLE
imputing a parent genotype

### DISCUSSION THREAD

**Discussion by Anilkumargpb** [AlphaGenes/AlphaPlantImpute2#8] [General]
Hi 

I have DH population which is genotyped and parents are not genotyped. I am looking to impute the parents genotype data using DH population genotype information. Do you have any examples in the repository to follow ?. Please let me know. 

Thanks 
Anil

**Comment by gregorgorjanc** [comment]
Have a look at examples in https://github.com/AlphaGenes/AlphaPlantImpute2/tree/main/example and let us know if they are not clear enough for you.

**Reply by Anilkumargpb** [reply]
Hi Gregor,

I have explored these example files. These files are examples for imputing population or imputing bi-parental cross. I am looking for imputing parents for which I don't have genotypes data. It is something like this https://acsess.onlinelibrary.wiley.com/doi/full/10.1002/csc2.20409. Sorry for troubling you. 

Thanks 
Anil

**Reply by gregorgorjanc** [reply]
I understand. Have you tried running an example where parents genotypes are set to missing? I think it should work.

**Reply by Anilkumargpb** [reply]
Thanks for the quick reply, I could not figure out which example to use and verify. Can you please specify?

Thanks

**Reply by gregorgorjanc** [reply]
Look at the examples.

Inspect how know and unknown/missing genotypes are encoded.

Implement the same for your example/setting, where progeny have known genotypes and parents have unknown/missing genotypes.

**Reply by Anilkumargpb** [reply]
Yeah, I was trying the same. I will write back to you after implementing it. 
Thank you so much for your time.
